CalibPPS/TimingCalibration/python/ALCARECOPPSTimingCalib_cff.py

```diff
@@ -0,0 +1,34 @@
+import FWCore.ParameterSet.Config as cms
+
+from RecoPPS.Configuration.recoCTPPS_cff import *
+from CalibPPS.TimingCalibration.ppsTimingCalibrationPCLWorker_cfi import ppsTimingCalibrationPCLWorker
+
+MEtoEDMConvertPPSTimingCalib = cms.EDProducer('MEtoEDMConverter',
+    Name = cms.untracked.string('MEtoEDMConverter'),
+    Verbosity = cms.untracked.int32(0),
+    Frequency = cms.untracked.int32(50),
+    MEPathToSave = cms.untracked.string('AlCaReco/PPSTimingCalibrationPCL'),
+    deleteAfterCopy = cms.untracked.bool(True),
+)
+
+# uncalibrated rechits/tracks
+ctppsDiamondRecHits.applyCalibration = False
```

Comment:
Can it be that I am modifying the reference for the whole chain here?

Reply:
Correct, this change applies to `ctppsDiamondRecHits` object that is the same object everywhere. Therefore it is strongly recommended to change python objects only in the files that define them. The proper way to have "local modification" is, as you already did, to `clone()` an object and modify the clone.

Reply:
Many thanks for this confirmation, @makortel! I'll be more careful in the future!